File: management/src/main/java/io/hexaforce/management/controller/ExaminationNumberTemporaryProcessController.java
package io.hexaforce.management.controller;

import java.util.Arrays;
import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;
import org.springframework.web.bind.annotation.RestController;

import io.hexaforce.dievas.commons.DievasServerMesseage;
import io.hexaforce.dievas.service.ExaminationNumberTemporaryProcessService;
import lombok.extern.slf4j.Slf4j;

/**
 * ExaminationNumberTemporaryProcess router function.
 * 
 * @version 1.0.0.BUILD-SNAPSHOT
 * @author T.Tantaka 
 */

@Slf4j
@RestController
@RequestMapping("/api")
public class ExaminationNumberTemporaryProcessController {
	
	@Autowired
	private ExaminationNumberTemporaryProcessService examinationNumberTemporaryProcessService;
	
	@ResponseBody
	@GetMapping(value = "/examination-number-temporary-process")
	public ResponseEntity<List<DievasServerMesseage>> init() {
		log.info("Controller(java): init()");
		String x = examinationNumberTemporaryProcessService.init();
		return new ResponseEntity<List<DievasServerMesseage>>(Arrays.asList(new DievasServerMesseage(x)),HttpStatus.OK);
	}
	
}

File: webapp/src/main/resources/static/controller/ExaminationNumberTemporaryProcessController.js
'use strict';
/*******************************************************************************
 * ExaminationNumberTemporaryProcessController
 ******************************************************************************/
app.controller('ExaminationNumberTemporaryProcessController', ['ExaminationNumberTemporaryProcessService', '$scope', function (ExaminationNumberTemporaryProcessService, $scope) {
  
  $scope.init = function () {
    console.log('Controller(js): init()');
    ExaminationNumberTemporaryProcessService.init();
    $scope.serviceName = ExaminationNumberTemporaryProcessService.getStorageData();
  }
  
}]);